event_reporter/reporter/application_event_reporter.go

```diff
@@ -87,13 +87,12 @@ func (s *applicationEventReporter) shouldSendResourceEvent(a *appv1.Application,
 	return true
 }
 
-func (r *applicationEventReporter) getDesiredManifests(ctx context.Context, a *appv1.Application, logCtx *log.Entry) (*apiclient.ManifestResponse, error, bool) {
+func (r *applicationEventReporter) getDesiredManifests(ctx context.Context, a *appv1.Application, logCtx *log.Entry) (*apiclient.ManifestResponse, bool) {
 	// get the desired state manifests of the application
 	project := a.Spec.GetProject()
 	desiredManifests, err := r.applicationServiceClient.GetManifests(ctx, &application.ApplicationManifestQuery{
 		Name:         &a.Name,
 		AppNamespace: &a.Namespace,
-		Revision:     &a.Status.Sync.Revision,
```

Comment:
why we removed revision from request?

Reply:
The request parameters for retrieving the manifest have changed. Previously, the revision was not passed in the request, and the manifest was always generated from the latest commit of the branch. Now the revision is passed, so we need to remove the revision from the function call to maintain the same behavior.